exercises/card_counting.livemd

```diff
@@ -41,27 +42,63 @@ Card values are as follows:
 When the current card is `4` and the current count is `0` what is the `next_count`?
 Replace `nil` with your answer.
 
+<!-- livebook:{"attrs":{"assertions":"assert next_count === 1","code":"card = 4\r\ncount = 0\r\n\r\nnext_count = nil","solution":"next_count = 1"},"kind":"Elixir.TestedCell","livebook_object":"smart_cell"} -->
```

Comment:
I'd like to encourage students to use arithmetic operators.

Can we change the text to:

When the current card is `4` and the current count is `0` what is the `next_count`?
Use arithmetic operators `+` or `-` with the `count` variable to set the `next_count`.

Then change the solution to:

```elixir
next_count = count + 1
```

Reply:
Yep I can do that :)